examples/ordered-options.mjs

```diff
@@ -0,0 +1,34 @@
+// Now might I enforce that two flags are specified in a specific order?
+
+import { parseArgs } from '../index.js';
+
+function findTokenIndex(tokens, target) {
+  return tokens.findIndex((token) => token.kind === 'option' &&
+    token.name === target
+  );
+}
+
+const experimentalName = 'enable-experimental-options';
+const unstableName = 'some-unstable-option';
+
+const options = {};
```

Comment:
```suggestion
const options = {
  [experimentalName]: { type: 'boolean' },
  [unstableName]: { type: 'boolean' },
};
```

Reply:
(Thanks. I stumbled when I was adding the kebab style options.)

Reply:
Done